documentation/basestyle.cls

```diff
@@ -22,7 +22,7 @@
 % Language.
 \RequirePackage{polyglossia}
 \setdefaultlanguage[variant=british]{english}
-\setotherlanguages{greek,german,russian,latin,french}
```

Comment:
Quousque tandem?

Reply:
For reasons that I do not want to investigate, Latin breaks _something_ we do:
```
Rotating Pulsating.tex
26
Undefined control sequence.
^->\active@prefix ^\@undefined 
                               
l.26 ...raph*{Derivative} The notation $\deriv x f
                                                  $ represents the evaluation
```

At the same time, we appear not to be using Latin, even where we have actual Latin, such as in cbrt.tex:
https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L222-L242 https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L1270-L1275 https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L1436-L1442

Contrast French, where we use `\textfrench`:
https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L337-L340